docker/pythonpath_dev/superset_config.py

```diff
@@ -28,18 +28,18 @@
 
 logger = logging.getLogger()
 
-DATABASE_DIALECT = os.getenv("DATABASE_DIALECT", "postgresql")
-DATABASE_USER = os.getenv("DATABASE_USER", "superset")
-DATABASE_PASSWORD = os.getenv("DATABASE_PASSWORD", "superset")
-DATABASE_HOST = os.getenv("DATABASE_HOST", "db")
-DATABASE_PORT = os.getenv("DATABASE_PORT", "5432")
-DATABASE_DB = os.getenv("DATABASE_DB", "superset")
-
-EXAMPLES_USER = os.getenv("EXAMPLES_USER", "examples")
-EXAMPLES_PASSWORD = os.getenv("EXAMPLES_PASSWORD", "examples")
-EXAMPLES_HOST = os.getenv("EXAMPLES_HOST", "db")
-EXAMPLES_PORT = os.getenv("EXAMPLES_PORT", "5432")
-EXAMPLES_DB = os.getenv("EXAMPLES_DB", "examples")
+DATABASE_DIALECT = os.getenv("DATABASE_DIALECT")
```

Comment:
I think this is acceptable. The only other formulation I can think is having Python parse the `.env-non-dev` file to read the fallbacks from there. It seems like the file adheres to the [configparser](https://docs.python.org/3/library/configparser.html) format.

Reply:
Spoke with @michael-s-molina offline who confirmed that the Docker variables are environment variables and thus my previous comment can be ignored.